.changeset/deep-ghosts-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ai-sdk/langchain': minor
+---
+
+[New Functionality] Support using data masking in LangChain orchestration client.
```

Comment:
I am not sure if we have to explicitly announce this in the release notes, as it worked out of the box.
I only added sample code and made no changes to the client.

Reply:
I don't mind if we call this a minor, as we never officially announced and tested this functionality.